diff --git Dockerfile Dockerfile
index 0c16092..7f24e3a 100644
--- Dockerfile
+++ Dockerfile
@@ -24,7 +24,7 @@ ARG PKG=github.com/lightningnetwork/lnd
 ARG TAGS_BASE="autopilotrpc signrpc walletrpc chainrpc invoicesrpc watchtowerrpc"
 
 # Added by yours truly (@lncm)
-ARG TAGS_EXTRA=""
+ARG TAGS_EXTRA="kvdb_etcd"
 
 
 #
@@ -255,3 +255,7 @@ EXPOSE  8080  9735  9911  10009
 
 # Specify the start command and entrypoint as the lnd daemon
 ENTRYPOINT ["/usr/local/bin/lnd"]
+
+# Make sure etcd variant has etcd support enabled on run, for more details see:
+#   https://github.com/lightningnetwork/lnd/blob/master/docs/etcd.md#configuring-lnd-to-run-on-etcd
+CMD ["--db.backend=etcd"]
