diff --git Dockerfile Dockerfile
index 929d281..d437a9c 100644
--- Dockerfile
+++ Dockerfile
@@ -125,7 +125,7 @@ ENV TAGS="autopilotrpc invoicesrpc walletrpc routerrpc watchtowerrpc"
 ENV TAGS="${TAGS} osusergo netgo static_build"
 
 # Added by yours truly (@lncm)
-ENV TAGS="${TAGS} wtclientrpc"
+ENV TAGS="${TAGS} wtclientrpc monitoring"
 
 
 # Install dependencies
@@ -179,7 +179,7 @@ ENV TAGS="autopilotrpc invoicesrpc walletrpc routerrpc watchtowerrpc"
 ENV TAGS="${TAGS} osusergo netgo static_build"
 
 # Added by yours truly (@lncm)
-ENV TAGS="${TAGS} wtclientrpc"
+ENV TAGS="${TAGS} wtclientrpc monitoring"
 
 RUN apt-get update \
     && apt-get -y install \
@@ -270,8 +270,11 @@ COPY  --from=cross-check  /bin/lncli  /bin/
 # Define a root volume for data persistence.
 VOLUME /root/.lnd
 
-# Expose lnd ports (rest, p2p, rpc respectively).
-EXPOSE  8080  9735  10009
+# Expose lnd ports (rest, monitoring, p2p, rpc respectively).
+EXPOSE  8080  8989  9735  10009
 
 # Specify the start command and entrypoint as the lnd daemon.
 ENTRYPOINT ["lnd"]
+
+# Okay to hardcode them here, as it's in a variant that specifically wants Prometheus
+CMD ["--prometheus.enable", "--prometheus.listen=0.0.0.0:8989"]
