diff --git Dockerfile Dockerfile
index ed9f858..485e8b5 100644
--- Dockerfile
+++ Dockerfile
@@ -26,7 +26,7 @@ ARG TAGS_LND="signrpc chainrpc"
 ARG TAGS_STATIC="osusergo netgo static_build"
 
 # Added by yours truly (@lncm)
-ARG TAGS_EXTRA="wtclientrpc"
+ARG TAGS_EXTRA="wtclientrpc monitoring"
 
 # Force Go to use the cgo based DNS resolver. This is required to ensure DNS
 #   queries required to connect to linked containers succeed
@@ -252,5 +252,11 @@ VOLUME /root/.lnd
 # Expose lnd ports (rest, p2p, rpc respectively)
 EXPOSE  8080  9735  10009
 
+# Expose monitoring/Prometheus port
+EXPOSE 8989
+
 # Specify the start command and entrypoint as the lnd daemon
 ENTRYPOINT ["lnd"]
+
+# Okay to hardcode them here, as it's in a variant that specifically wants Prometheus
+CMD ["--prometheus.enable", "--prometheus.listen=0.0.0.0:8989"]
