diff --git Dockerfile Dockerfile
index cda3a7e..641f9c2 100644
--- Dockerfile
+++ Dockerfile
@@ -29,7 +29,7 @@ ARG TAGS_LND="signrpc chainrpc"
 ARG TAGS_STATIC="osusergo netgo static_build"
 
 # Added by yours truly (@lncm)
-ARG TAGS_EXTRA=""
+ARG TAGS_EXTRA="experimental"
 
 # Force Go to use the cgo based DNS resolver. This is required to ensure DNS
 #   queries required to connect to linked containers succeed
