diff --git Dockerfile Dockerfile
index c3f2274..91b48fa 100644
--- Dockerfile
+++ Dockerfile
@@ -147,8 +147,11 @@ COPY  --from=cross-check  /bin/lncli  /bin/
 # Define a root volume for data persistence.
 VOLUME /root/.lnd
 
-# Expose lnd ports (rest, p2p, rpc respectively).
-EXPOSE 8080 9735 10009
+# Expose lnd ports (rest, monitoring, p2p, rpc respectively).
+EXPOSE 8080 8989 9735 10009
 
 # Specify the start command and entrypoint as the lnd daemon.
 ENTRYPOINT ["lnd"]
+
+# Okay to hardcode them here, as it's in a variant that specifically wants Prometheus
+CMD ["--prometheus.enable", "--prometheus.listen=0.0.0.0:8989"]
diff --git build.sh build.sh
index 3da6714..0f73bc8 100755
--- build.sh
+++ build.sh
@@ -43,6 +43,9 @@ TAGS="autopilotrpc invoicesrpc walletrpc routerrpc watchtowerrpc"
 # TODO: Check if still needed after Go v1.14 release
 TAGS="${TAGS} osusergo netgo static_build"
 
+# Added by yours truly (@lncm) to enable monitoring
+TAGS="${TAGS} monitoring"
+
 build() {
   binary_name=$1
   extra_tags=$2
