diff --git Dockerfile Dockerfile
index 88899c3..14088e7 100644
--- Dockerfile
+++ Dockerfile
@@ -146,8 +146,11 @@ COPY  --from=cross-check  /bin/lncli  /bin/
 # Define a root volume for data persistence.
 VOLUME /root/.lnd
 
-# Expose lnd ports (rest, p2p, rpc respectively).
-EXPOSE 8080 9735 10009
+# Expose lnd ports (rest, monitoring, p2p, rpc respectively).
+EXPOSE 8080 8989 9735 10009
 
 # Specify the start command and entrypoint as the lnd daemon.
 ENTRYPOINT ["lnd"]
+
+# Okay to hardcode them here, as it's in a variant that specifically wants Prometheus
+CMD ["--prometheus.enable", "--prometheus.listen=0.0.0.0:8989"]
diff --git build.sh build.sh
index eceaf46..ea122e5 100755
--- build.sh
+++ build.sh
@@ -33,6 +33,9 @@ TAGS="autopilotrpc invoicesrpc walletrpc routerrpc watchtowerrpc"
 # TODO: Check if still needed after Go v1.14 release
 TAGS="${TAGS} osusergo netgo static_build"
 
+# Added by yours truly (@lncm) to enable monitoring
+TAGS="${TAGS} monitoring"
+
 
 build() {
   binary_name=$1
