diff --git Dockerfile Dockerfile
index 3ca30a1..aae0ffd 100644
--- Dockerfile
+++ Dockerfile
@@ -23,7 +23,7 @@ ARG TAGS_LND="signrpc chainrpc"
 ARG TAGS_STATIC="osusergo netgo static_build"
 
 # Added by yours truly (@lncm)
-ARG TAGS_EXTRA=""
+ARG TAGS_EXTRA="monitoring"
 
 
 
@@ -273,5 +273,11 @@ VOLUME /root/.lnd
 # Expose lnd ports (rest, p2p, watchtower, rpc respectively).
 EXPOSE 8080 9735 9911 10009
 
+# Expose monitoring/Prometheus port
+EXPOSE 8989
+
 # Specify the start command and entrypoint as the lnd daemon.
 ENTRYPOINT ["lnd"]
+
+# Okay to hardcode them here, as it's in a variant that specifically wants Prometheus
+CMD ["--prometheus.enable", "--prometheus.listen=0.0.0.0:8989"]
